lms/djangoapps/ora_staff_grader/serializers.py

```diff
@@ -139,6 +139,7 @@ class UploadedFileSerializer(serializers.Serializer):
     downloadUrl = serializers.URLField(source='download_url')
     description = serializers.CharField()
     name = serializers.CharField()
+    size = serializers.IntegerField()
```

Comment:
Nit: from enough security schooling, probably worth adding a min value (unless we expect sizes < 0...)

```suggestion
    size = serializers.IntegerField(min_value=0)
```

Reply:
@jansenk ^